zoo-css-challenge/style.css

```diff
@@ -28,9 +37,13 @@ section {
   padding: 5%;
   background-color: #f8f0fb;
 }
+/*background color to white*/
+.introduction{
+  background-color: white;
+}
 
 img {
-  border: 5px solid #8075ff;
+  border:0;
```

Comment:
We could also delete this `img` rule because `img` elements have no border by default